src/EndpointManager.sol

```diff
@@ -476,39 +561,94 @@ abstract contract EndpointManager is
 
         address transferRecipient = bytesToAddress(nativeTokenTransfer.to);
 
+        // Check inbound rate limits
+        bool isRateLimited = _isInboundAmountRateLimited(nativeTransferAmount, message.chainId);
+        if (isRateLimited) {
+            // queue up the transfer
+            uint64 queueSequence = _useInboundQueueSequence();
+
+            _getInboundQueueStorage()[queueSequence] = InboundQueuedTransfer({
+                amount: nativeTransferAmount,
+                recipient: transferRecipient,
+                txTimestamp: block.timestamp
+            });
+
+            emit InboundTransferQueued(queueSequence);
```

Comment:
I'm just wondering whether from a monitoring point of view we want to include the sending chain id (i.e. the chain id of the origin chain) in this event?